spring-integration-sftp/src/main/java/org/springframework/integration/sftp/session/DefaultSftpSessionFactory.java

```diff
@@ -351,7 +352,12 @@ private ClientSession initClientSession() throws IOException {
 						.verify(verifyTimeout)
 						.getSession();
 
-		clientSession.auth().verify(verifyTimeout);
+		try {
+			clientSession.auth().verify(verifyTimeout);
+		} catch (IOException e) {
```

Comment:
Our style is `catch` on a new line.
Please, run `gradlew :spring-integration-sftp:check` locally before pushing.